llama.cpp

```diff
@@ -9465,6 +9484,7 @@ float * llama_get_logits(struct llama_context * ctx) {
 }
 
 float * llama_get_logits_ith(struct llama_context * ctx, int32_t i) {
+    assert(ctx->logits_valid.at(i));
```

Comment:
Should this be:

```suggestion
    assert(ctx->logits_valid.at(i*ctx->model.hparams.n_vocab));
```

?